openapi/paths/invoices@{id}@credit-memo-allocations.yaml

```
@@ -5,16 +5,18 @@ get:
     - Core
   tags:
     - Invoices
-  summary: Get credit memo allocations to invoices
+  summary: Retreive credit memo invoice allocations
   operationId: GetInvoiceCreditMemoAllocationCollection
   x-sdk-operation-name: getAllCreditMemoAllocations
-  description: Get the precise amounts from a credit memo allocated to the invoice.
+  description: |-
+    Retrieves the precise amounts that have been allocated from a
```

Nitpick: I think `precise` doesn't add anything here. It makes me wonder if all other amounts are imprecise.
```suggestion
    Retrieves the amounts that have been allocated from a
```